### DISCUSSION TITLE
유니티에서 제페토 게임예제만드는 실행창이 사라졌어요

### DISCUSSION THREAD

**Discussion by SaesolPark** [naverz/zepeto-studio-kor#828] [5. Q&A]
![유니티 오류](https://user-images.githubusercontent.com/80307607/171349307-00e8d8fc-c6f9-4160-8500-62e6ac5c9fae.png)
위 사진처럼 갑자기 저장 다 하고 컴퓨터 껐다가 켰더니 오류가 나면서 제페토 실행창이 사라졌어요
이럴땐 어떻게 해결해야 하나요? 
재질이랑 라이팅 다 입힌 상태라 **잃고싶지 않아요** 살려주세요 제발

**Comment by x4cf** [comment]
레이아웃 변경으로 UI만 가려진것같습니다. 리임포트올을 한번 해주세요 :)
<img width="608" alt="리임포트올" src="https://user-images.githubusercontent.com/90679926/171353882-e4d4ae95-0ac3-42da-8e76-7420b39c739f.png">

**Reply by SaesolPark** [reply]
감사합니다 해볼게요♥

**Reply by SaesolPark** [reply]
리임포트올해도 안생기네요ㅜㅜㅜ
